tasks/setup-Debian.yml

```diff
@@ -13,5 +13,21 @@
     state: present
     update_cache: yes
 
+- name: Download specific Jenkins version.
+  get_url:
+    url: "{{ jenkins_pkg_url }}/jenkins_{{ jenkins_version }}_all.deb"
+    dest: "/tmp/jenkins.deb"
+  when: jenkins_version is defined
+
+- stat: path="/tmp/jenkins.deb"
```

Comment:
This needs a `name` on it, like `Check if a specific version of Jenkins is currently installed.`.